
package logica.servidor;

import jakarta.xml.ws.WebFault;


/**
 * This class was generated by the XML-WS Tools.
 * XML-WS Tools 4.0.0
 * Generated source version: 3.0
 * 
 */
@WebFault(name = "AsignarOrdenAOfertaFinalizada",  targetNamespace = "http://servidor.logica/")
public class AsignarOrdenAOfertaFinalizada_Exception
    extends Exception
{

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private AsignarOrdenAOfertaFinalizada faultInfo;

    /**
     * 
     * @param faultInfo
     * @param message
     */
    public AsignarOrdenAOfertaFinalizada_Exception(String message,  AsignarOrdenAOfertaFinalizada faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param cause
     * @param faultInfo
     * @param message
     */
    public AsignarOrdenAOfertaFinalizada_Exception(String message,  AsignarOrdenAOfertaFinalizada faultInfo,  Throwable cause) {
        super(message,  cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *     returns fault bean: logica.servidor.AsignarOrdenAOfertaFinalizada
     */
    public AsignarOrdenAOfertaFinalizada getFaultInfo() {
        return faultInfo;
    }

}
